include/neml2/base/config.h.in

```diff
@@ -39,4 +33,4 @@
 /// The tolerance used in various algorithms
```

Comment:
Shouldn't these be (global) options now?

Reply:
I think you are right, now that dtype is a run-time thing. I will make them also global options.